app/controllers/users.js

```diff
@@ -0,0 +1,32 @@
+const User = require('../models').user,
+  errors = require('../errors');
+
+exports.create = (req, res, next) => {
+  const user = req.body
+    ? {
```

Comment:
there is no need to do this validation for existence in the body since you are already validating the parameters